q2_composition/plugin_setup.py

```diff
@@ -40,6 +40,18 @@
     artifact_format=BIOMV100DirFmt
 )
 
+plugin.register_semantic_types(Balance)
+
+plugin.register_semantic_type_to_format(
+    FeatureTable[Balance],
+    artifact_format=BIOMV210DirFmt
+)
+
+plugin.register_semantic_type_to_format(
+    FeatureTable[Balance],
+    artifact_format=BIOMV100DirFmt
+)
```

Comment:
You should drop this registration (and the similar one for `Composition` to `BIOMV100DirFmt`). It doesn't actually do anything, it's still left-over from `q2-types` for a situation where we abused the framework's lookup for a dynamic unit-test (none of those were *great* ideas, but it worked well enough at the time).